replication-api/src/main/java/com/connexta/ion/replication/api/Replicator.java

```diff
@@ -22,6 +22,8 @@
    * Submits a {@link SyncRequest} to be executed
    *
    * @param syncRequest The {@link SyncRequest} to run
+   *
+   * @throws InterruptedException on error inserting request into the queue
```

Comment:
❗️ This is not why `InterruptedException` would be thrown. It would be thrown because the thread was interrupted while trying to insert the request into the queue.

Reply:
Updated